docs/sources/static/api/_index.md

```diff
@@ -37,6 +37,12 @@ The following endpoints are exposed:
 - Update config: [`PUT /agent/api/v1/config/{name}`](#update-config)
 - Delete config: [`DELETE /agent/api/v1/config/{name}`](#delete-config)
 
+{{< admonition type="note" >}}
+If you are running Grafana Agent in a Docker container, you must change the default HTTP listen address from `127.0.0.0:12345` to `0.0.0.0:12345` to expose the API outside the Docker container.
```

Comment:
Technical nit that might take away from the clarity of this information:

It doesn't have to be the IP address `0.0.0.0`, it just needs to be the address of a network interface other than the loopback.

I don't actually know off the top of my head a convenient and universal way to get that information in the container so it probably is best to just use `0.0.0.0`.
It's certainly what I'd do.

The following suggestion corrects on the default listen address (it's `127.0.0.1`).
```suggestion
If you are running Grafana Agent in a Docker container, you must change the default HTTP listen address from `127.0.0.1:12345` to `0.0.0.0:12345` to expose the API outside the Docker container.
```

Although the process is now listening on the container's network interface, it's not necessarily accessible from the host.
You also need to publish the port with your container manager.
For example, with Docker, https://docs.docker.com/network/#published-ports.

Reply:
Might be well worth adding the extra bit about publishing the port.... I'll rework this a bit.